Sources/SwiftProtobuf/Enum.swift

```diff
@@ -16,19 +16,26 @@
 // -----------------------------------------------------------------------------
 
 /// Generated enum types conform to this protocol, which provides the
-/// hashability requirement for enums as well as the name mapping requirement
```

Comment:
Here I removed the prose about the name mapping requirement because it's not part of the public API, so it doesn't need to be in the public docs.